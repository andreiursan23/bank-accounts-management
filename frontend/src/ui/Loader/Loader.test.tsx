import { render } from '../../../test/utilities'
import Loader from './Loader'

describe('Loader component', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<Loader />)

    expect(asFragment()).toMatchSnapshot()
  })
})
