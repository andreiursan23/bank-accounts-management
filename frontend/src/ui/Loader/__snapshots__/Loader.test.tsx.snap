// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loader component > matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="w-10 h-10 bg-[#4768fa]"
    data-testid="loader"
  />
</DocumentFragment>
`;
